locations/spiders/kinleigh_folkard_hayward_gb.py

```
@@ -0,0 +1,26 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class KinleighFolkardHaywardGBSpider(SitemapSpider, StructuredDataSpider):
+    name = "kinleigh_folkard_hayward_gb"
+    item_attributes = {"brand": "Kinleigh Folkard & Hayward", "country": "GB"}
+    sitemap_urls = ["https://www.kfh.co.uk/sitemap-pages.xml"]
+    sitemap_rules = [(r"^https://www\.kfh\.co\.uk/branch-finder/[^/]+$", "parse_sd")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)